core/src/main/java/io/aiven/klaw/service/ClusterApiService.java

```diff
@@ -585,6 +585,32 @@ public ApiResponse getAivenServiceAccountDetails(
     }
   }
 
+  public ApiResponse getAivenServiceAccounts(String projectName, String serviceName, int tenantId)
+      throws KlawException {
+    getClusterApiProperties(tenantId);
+    try {
+      String uriGetServiceAccounts =
+          clusterConnUrl + "/topics/serviceAccounts/project/projectName/service/serviceName";
+      uriGetServiceAccounts =
```

Comment:
I don't know if we will ever need to use it again in the future but I think putting path as a class constant would be a good convention?

Reply:
Good one. Have created an issue #440  to move all cluster api endpoints to some constants class. I see several hard codings which is not looking good.